app/models/order.rb

```diff
@@ -0,0 +1,18 @@
+class Order < ApplicationRecord
+  enum status: [:pending, :completed]
+  
+  belongs_to :user
+  has_many :order_items
+  
+  def add_order_items(cart) 
+    cart.order_items.each do |item|
+      item.cart_id = nil
+      order_items << item
+    end 
+  end
+
+  def total_price
+    order_items.to_a.sum { |item| item.total_price }
```

Comment:
you could have change it here as well :D

Reply:
forgot, my bad <3. fixed. ✓